ct/client/main/main.go

```diff
@@ -8,17 +8,14 @@ import (
 	"encoding/json"
 	ct "github.com/zmap/zcrypto/ct"
 	"github.com/zmap/zcrypto/ct/client"
-	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/ct/x509"
```

Comment:
I did this because it's currently what ctsync pull_daemon uses to parse certificates. Since I was using this to figure out what pull_daemon was missing, this is why I modified it.